discover/portal/articles/210-setting-up/00-intro.markdown

```diff
@@ -0,0 +1,90 @@
+# Setting Up [](id=setting-up)
+
+If @product@ is anything, it's configurable. As the core is shrinking
+due to its increased modularity, it's important that all the applications in
+Liferay are also configurable. 
+
+Breaking it down, three types of applications must be configurable:
+
+1.  The portal itself
+2.  Liferay's native applications 
+3.  Custom applications
+
+To this end, Liferay's engineers have made the platform and its own applications
+configurable, and created a mechanism for developers to make their
+[applications configurable](/develop/tutorials/-/knowledge_base/7-1/configurable-applications).
+
+## Where Configuration Happens [](id=where-configuration-happens)
+
+Liferay's configuration takes place in the following places: 
+
+Properties files: properties files  that set default behavior may be included in
+the platform or the modules. Keep in mind that these settings can always be
+overwritten by a system administrator in the UI. To find what properties are
+configurable this way, visit 
+[docs.liferay.com](https://docs.liferay.com/portal/7.1-latest/propertiesdoc).
+
+Database: configuration through Liferay's UI is stored in a database.  The
+values in the database always override the default configurations set through
+properties files.
+
+The UI location where these database configuration options appear depends on the
+scope you want to affect with the settings you choose.
+
+## Configuration Scope [](id=configuration-scope)
+
+Depending on the configuration scope of a setting you change, you'll impact
+Liferay and its applications with more or less granularity. At one end of the
+spectrum, you can affect the scope of the whole *System*. Configurations made at
+the System scope affect all portal instances, sites, and portlet instances. At
```

Comment:
Instead of "portlet instances" we are now calling it "Widget Scope" in the System Settings UI

Reply:
This applies to every time "portlet instance" is used thorough the article, not just here.